h2o-py/tests/testdir_misc/pyunit_gh_15614_datatable_2_pandas.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+try:
+    import datatable
+except ImportError:
+    pyunit_utils.install("datatable")
+from h2o.utils.shared_utils import (can_use_datatable, can_use_pandas)
+import time
+import random
+
+# if datable is installed, this test will show that using datatable to convert h2o frame to pandas frame is
+# much faster.  This workaround is suggested by Megan Kurka.  I implemented the code snippet from Bernard Ong.
+def test_datatable_pandas():
+    if can_use_pandas() and can_use_datatable():    # only perform tests if pandas and datatable are installed
+        h2oFrame = h2o.import_file(pyunit_utils.locate("smalldata/glm_test/multinomial_3Class_10KRow.csv"))
+        # convert frame not using datatable
+        startT = time.time()
+        original_pandas_frame = h2oFrame.as_data_frame()
+        oldTime = time.time()-startT
+        print("H2O frame to Pandas frame conversion time: {0}".format(oldTime))
+        # convert frame using datatable
+        startT = time.time()
+        new_pandas_frame = h2oFrame.as_data_frame(multi_thread=True, use_datatable=True)
```

Comment:
This will likely fail due to `use_datatable` keyword argument - it's not implemented anymore. I'd remove this test.